crates/fuel-core/src/combined_database.rs

```diff
@@ -523,6 +523,14 @@ impl CombinedDatabase {
 
         Ok(())
     }
+
+    pub fn shutdown(self) {
```

Comment:
This seems unrelated to the main PR. Does this have a specific purpose compared to just dropping the handles? I'm fine with doing this here, but maybe the changelog should mention it?

Reply:
cc @xgreenx that added this change